File: backend/booking-service/src/main/java/com/project/booking/dto/booking/BookingPassengerInfoResponse.java
package com.project.booking.dto.booking;

import java.time.LocalDateTime;
import java.util.UUID;

public record BookingPassengerInfoResponse(
        Long id,
        String fullName,
        String gender,
        String phoneNumber,
        String email,
        UUID roomId,
        String checkinStatus,
        LocalDateTime checkedInAt) {
}

File: frontend-android/app/src/main/java/com/project/cruise/android/data/dto/booking/BookingPassengerInfoResponse.kt
package com.project.cruise.android.data.dto.booking

data class BookingPassengerInfoResponse(
    val id: Long?,
    val fullName: String?,
    val gender: String?,
    val phoneNumber: String?,
    val email: String?,
    val roomId: String?,
    val checkinStatus: String?,
    val checkedInAt: String?
)